js/script.js

```diff
@@ -72,4 +72,46 @@ window.onload = () => {
   myBookLib.bookList.forEach((book) => {
     UI(book);
   });
-};
\ No newline at end of file
+};
+
+function navigation(section) {
+  const List = document.querySelector('#show-book');
+  const Form = document.querySelector('.form');
+  const Contact = document.querySelector('.contact');
+  const heading = document.querySelector('.header');
+
+  switch (section) {
+    case 'show-book':
+      List.style.display = 'block';
+      Form.style.display = 'none';
+      Contact.style.display = 'none';
+      heading.innerHTML = 'All Awesome Books';
+      break;
+
+    case 'form':
+      List.style.display = 'none';
+      Form.style.display = 'block';
+      Contact.style.display = 'none';
+      heading.innerHTML = 'Add a New Book';
+      break;
+
+    case 'contact':
+      List.style.display = 'none';
+      Form.style.display = 'none';
+      Contact.style.display = 'block';
+      heading.innerHTML = 'Contact Information';
+      break;
+
+    default: break;
+  }
+}
+
+// function setDate() {
+//   const date = document.getElementById('date');
+//   const { DateTime } = luxon;
+//   date.innerHTML = DateTime.now().toFormat('LLL dd yyyy, t');
+// }
+
+const dateContainer = document.querySelector('.date');
+dateContainer.innerHTML = new Date().toString().slice(0, 34);
+// bookObj.loadBooks();
```

Comment:
- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code, this will make your codebase look neat and readable 😉